src/channels/components/ChannelDeleteDialog/ChannelDeleteDialog.tsx

```diff
@@ -75,21 +79,26 @@ const ChannelDeleteDialog: React.FC<ChannelDeleteDialogProps> = ({
   return (
     <ActionDialog
       confirmButtonState={confirmButtonState}
+      backButtonText={
+        canBeDeleted
+          ? buttonMessages.cancel.defaultMessage
+          : buttonMessages.ok.defaultMessage
+      }
       open={open}
       onClose={onClose}
       onConfirm={() => (canBeDeleted ? onConfirm(choice) : onBack())}
-      title={intl.formatMessage(messages.deleteChannel)}
+      title={intl.formatMessage(messages.deleteChannel, { channelSlug })}
       confirmButtonLabel={intl.formatMessage(
-        canBeDeleted ? buttonMessages.delete : buttonMessages.ok,
+        canBeDeleted ? buttonMessages.delete : buttonMessages.cancel,
       )}
-      variant={canBeDeleted ? "delete" : "default"}
+      variant={canBeDeleted ? "delete" : "info"}
     >
       <div>
         {hasOrders ? (
           hasChannels ? (
             <>
               <Typography>
-                {intl.formatMessage(messages.needToBeMoved)}
+                {intl.formatMessage(messages.needToBeMoved, { br: <br /> })}
```

Comment:
issue: please create two lines of translations and don't put any html tags as variable to it